TestWebService/Package.resolved.license

```diff
@@ -0,0 +1,7 @@
+                  
```

Comment:
As noted in the other Package.resolved file it would make sense to remove it from the REUSE checks